engines/savestate.cpp

```diff
@@ -95,7 +95,14 @@ bool SaveStateDescriptor::isAutosave() const {
 
 bool SaveStateDescriptor::hasAutosaveName() const
 {
-	return _description.contains(_("Autosave"));
+	Common::U32String autosave = _("Autosave");
+	if (_description.size() >= autosave.size())
+		return _description.contains(autosave);
+	if (autosave.size() < 14)
+		return autosave == _description;
```

Comment:
I don't know if any engines append a number at the end of the autosave? We could also do

`_description.substr(0, autosave.size()) == autosave;`